src/Debug/TraceableDomainEventDispatcher.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Biig\Component\Domain\Debug;
+
+use Biig\Component\Domain\Event\DelayedListener;
+use Biig\Component\Domain\Event\DomainEvent;
+use Biig\Component\Domain\Event\DomainEventDispatcher;
+use Biig\Component\Domain\Event\DomainEventDispatcherInterface;
+use Biig\Component\Domain\Exception\InvalidArgumentException;
+use Biig\Component\Domain\Model\ModelInterface;
+use Biig\Component\Domain\Rule\DomainRuleInterface;
+use Biig\Component\Domain\Rule\PostPersistDomainRuleInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher;
+use Symfony\Component\EventDispatcher\Debug\WrappedListener;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class TraceableDomainEventDispatcher extends TraceableEventDispatcher implements DomainEventDispatcherInterface
+{
+    /**
+     * @var array
+     */
+    private $eventsFired;
+
+    /**
+     * @var DomainEventDispatcher
+     */
+    private $decorated;
+
+    /** @var array */
```

Comment:
Please use same form of comment than everywhere in this project.

Reply:
Yes I wil